tests/test_repository_tool.py

```diff
@@ -1349,10 +1368,20 @@ def test_delegate(self):
         self.targets_object.delegate, rolename, public_keys, paths, threshold,
         list_of_targets, path_hash_prefixes)
 
-    # Test non-existent target paths.
-    self.assertRaises(securesystemslib.exceptions.Error,
-        self.targets_object.delegate, rolename, public_keys, [], threshold,
-        ['/non-existent'], path_hash_prefixes)
+    # A path or target starting with a directory separator
+    self.assertRaises(tuf.exceptions.InvalidNameError,
+        self.targets_object.delegate, rolename, public_keys, ['/*'])
+    self.assertRaises(tuf.exceptions.InvalidNameError,
+        self.targets_object.delegate, rolename, public_keys, [],
+        list_of_targets=['/file1.txt'])
+
+    # A path or target starting with a directory separator
```

Comment:
This comment doesn't seem right, aren't we testing Windows path separators here?

Reply:
Fixed in squashed commit https://github.com/theupdateframework/tuf/pull/1008/commits/3a13b7312a9148f7aac553affa371f3f26c69f11